pkg/websocket/payload.go

```diff
@@ -27,6 +29,15 @@ type OutgoingPayload struct {
 	Params   map[string]interface{} `json:"params,omitempty"`
 }
 
+func (p *OutgoingPayload) ChannelUUID() string {
+	// "https://flows.stg.cloud.weni.ai/c/wwc/c4dc40fa-37e0-4147-a379-a3e8ffd23f80/receive"
```

Comment:
Remove this comment?

Reply:
I would like to keep it for later reference